css/templates/e-commerce/page.css

```diff
@@ -193,8 +193,26 @@ amp-selector [option][selected] {
   outline-color: var(--color-primary);
 }
 
+amp-carousel {
+  padding-left: 40px;
+  padding-right: 40px;
+}
+
 .amp-carousel-button {
-  display: none;
+  background: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="30px" height="30px" viewBox="0 0 24 24" fill="#000000"><path d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"/><path d="M0-.5h24v24H0z" fill="none"/></svg>') 30px 30px transparent;
```

Comment:
Overriding arrow icon in https://github.com/ampproject/ampstart/blob/master/components/carousel/carousel.css#L19